kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/players/QueuedAudioPlayer.kt

```diff
@@ -73,11 +73,9 @@ class QueuedAudioPlayer(context: Context, bufferConfig: BufferConfig? = null) :
     /**
      * Add a single item to the queue.
      * @param item The [AudioItem] to add.
-     * @param playWhenReady If this is `true` it will automatically start playback. Default is `true`.
+     * @param playWhenReady If the AudioPlayer has no item loaded, it will load the `item`. If this is `true` it will automatically start playback. Default is `true`.
      */
     fun add(item: AudioItem, playWhenReady: Boolean = true) {
-        exoPlayer.playWhenReady = playWhenReady
```

Comment:
Any reason why this was removed? Also, we still have it as a parameter.

Reply:
you're calling it twice, once here and at the end of the method.

Reply:
Oops 😬